src/modules/notes/controllers/deleteNote.js

```diff
@@ -0,0 +1,3 @@
+export const deleteNote = (req, res) => {
+	return res.status(201).json({success: true, value: req.params.id});
```

Comment:
Could you please move all status codes to the common constant variable and reuse them

Reply:
done